crates/sui-graphql-e2e-tests/tests/stable/prune/prune.exp

```diff
@@ -70,3 +70,55 @@ Response: {
     }
   }
 }
+
+task 12, lines 57-67:
+//# run-graphql
+Response: {
+  "data": {
+    "availableRange": {
+      "first": null,
```

Comment:
This does not seem right. This should be the sequence number of the first available checkpoint, how can it be ...unavailable?

Reply:
good catch, looking

Reply:
@amnn turns out it's b/c the available range (max_snap_cp, max_cp), while the pruning does not handle the case when max_snap_cp is already pruned, I fixed that in the pruner codes although this is unlikely in production as one epoch will have more checkpoints than the consistency range checkpoints, it may be worth adding the guardian codes to `indexer-alt` too if that makes sense.